src/main/frontend/Admin.html

```diff
@@ -289,7 +343,9 @@ <h1>Flights Database</h1>
 
         if (confirmation) {
             // User confirmed, proceed with deletion
-            fetchData(pagenumber, pagesize, sortOrder);
+            let url = `http://127.0.0.1:5000/scene/delete?DeleteOlder=${document.getElementById("DateOlder").value}`;
+            fetch(url);
+            sleep(2000).then(() => { fetchData(pagenumber, pagesize, sortOrder)});
```

Comment:
Try to actualy use the timeouts and async stuff. Hardcoding a sleep event is a wierd idea when javascript has a lot of things for this